packages/grafana-ui/src/components/Logs/LogLabels.tsx

```diff
@@ -12,6 +12,7 @@ interface Props {
   labels: Labels;
 }
 
+/** @deprecated */
```

Comment:
I'd add something like 
```suggestion
/** @deprecated  will be removed in the next major version */
```